sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -517,6 +517,45 @@ def __init__(self, message, response, parts):
         super(PartialBatchErrorException, self).__init__(message=message, response=response)
 
 
+class BatchErrorException(HttpResponseError):
```

Comment:
Seems that we have one of these in `_error.py` and one here? What's the distinction?